src/pages/BrowserPage.qml

```diff
@@ -141,12 +143,37 @@ Page {
         rotationHandler: browserPage
     }
 
+    function inputMaskForOrientation(orientation) {
+        // mask is in portrait window coordinates
+        var mask = Qt.rect(0, 0, Screen.width, Screen.height)
+        if (webView.enabled && browserPage.active && !webView.popupActive) {
+            var overlayVisibleHeight = browserPage.height - overlay.y
+
+            switch (orientation) {
+            case Orientation.None:
+            case Orientation.Portrait:
+                mask.y = overlay.y
+                // fallthrough
+            case Orientation.InvertedPortrait:
+                mask.height = overlayVisibleHeight
+                break
```

Comment:
This doesn't work correctly. 

Input ends up being (x,y,w,h):  0 0 1536 2048. Thus, web content cannot be used.